ml/README.md

```diff
@@ -6,21 +6,30 @@ This is a simple application of Convolution Neural Networks combined with backgr
 
 ## Requirements
 
-* Python 3
-* Tensorflow (1.14.0)
-* TfLearn (0.3.2)
-* Opencv Python (4.1.1.26)
-* Numpy (1.15.0)
-* Pillow (PIL) (6.1.0)
-* Imutils (0.5.3)
+- Python 3
+- Tensorflow (1.14.0)
+- TfLearn (0.3.2)
+- Opencv Python (4.1.1.26)
+- Numpy (1.15.0)
+- Pillow (PIL) (6.2.0)
+- Imutils (0.5.3)
+
+## Installation
+
+`requirements.txt` specifies the required packages needed to run this project.
+To install these packages, you must have **pip** installed on your machine.
+
+```bash
+pip install -r requirements.txt
```

Comment:
`pip` should be `pip3` since we will be using python3.

Reply:
Yes, I had pip3 alias set to pip on my machine. I forgot I did that.